### PR TITLE
Add RemittanceAPI

### PR DESCRIPTION
**Pull request by JagdeepSingh**
- It adds support to register payment accounts and deposits.
- It also adds error messages to GMO API response to make response understand better.

I added a new wiki page for the documentation [here](https://github.com/t-k/gmo-payment-ruby/wiki/Remittance-API).

### CODE REVIEW

File: lib/gmo.rb [hunk 1]
```diff
@@ -94,6 +95,10 @@ class ShopAndSiteAPI < API
       include ShopAndSiteAPIMethods
     end
 
+    class AccountAPI < API
```
Comment (johnnyshields) [inline lib/gmo.rb]: Maybe this should be called "DepositAPI" instead?

**Comment by johnnyshields** [comment]
@t-k 様、このPRはGMOの振り込みAPIです。銀行口座の登録も可能です。

**Comment by johnnyshields** [comment]
@t-k this is ready to merge, please review.

**Comment by t-k** [comment]
@JagdeepSingh @johnnyshields  Thanks for PR!
Please filter values you added on [`spec/support/config.yml`](https://github.com/JagdeepSingh/gmo-payment-ruby/commit/3051faf184d49b7319f1fa8a5cc10264a0bf3d9d#diff-448ef08af2b6f31b88461364e42c7b83) , through [`spec/support/vcr.rb`](https://github.com/t-k/gmo-payment-ruby/blob/master/spec/support/vcr.rb).

https://relishapp.com/vcr/vcr/v/3-0-3/docs/configuration/filter-sensitive-data

**Comment by johnnyshields** [comment]
@JagdeepSingh I think maybe we should rename this "RemittanceApi" because that's what GMO calls it based on domain name.

**Comment by JagdeepSingh** [comment]
> I think maybe we should rename this "RemittanceApi" because that's what GMO calls it based on domain name.

@johnnyshields I did the rename.

**Comment by JagdeepSingh** [comment]
> Please filter values you added on spec/support/config.yml , through spec/support/vcr.rb.

@t-k i have filtered the sensitive data in rspecs.

**Pull request merged**